_templates/exchanges.html

```diff
@@ -168,7 +169,8 @@ <h3 id="nepal"><img src="/img/flags/NP.png" alt="Nepalese flag">Nepal</h3>
     <div>
       <h3 id="new-zealand"><img src="/img/flags/NZ.png" alt="New Zealand Flag">New Zealand</h3>
       <p>
-        <a href="https://kiwi-coin.com/">Kiwi-coin</a>
+        <a href="https://kiwi-coin.com/">Kiwi-coin</a><br>
+        <a href="https://www.independentreserve.com/">Independent Reserve</a>
```

Comment:
We list the exchanges alphabetically under each country - could you please fix this placement, here?